index.js

```diff
@@ -8,11 +8,12 @@ if (!uuid) {
   NSUserDefaults.standardUserDefaults().setObject_forKey(uuid, kUUIDKey)
 }
 
-var variant = MSApplicationMetadata.metadata().variant;
+var sketchVersion = Settings.version.sketch
+var variant = sketchVersion >= 72 ? BCSketchInfo.shared().metadata().variant : MSApplicationMetadata.metadata().variant
```

Comment:
I forgot about that, maybe it'd be an interesting info to surface in `sketch/Settings`

Reply:
Definitely, we’ll need to think about that.